### ISSUE TITLE
Issue with PMTCT_STAT and TB_STAT known positives

### ISSUE THREAD

**Issue by noahbartlett** [pepfar-datim/Data-Pack-Feedback#41]
**Describe the bug**
The way known positives are calculated assumes that there are X positives therefore (X/yield - X) negatives and everyone else is a previously known positive. Obviously, this greatly exaggerates the number of known positives. 

**To Reproduce**
TB_STAT_ART columns L M and N
PMTCT_STAT column J K and L

**Expected behavior**
What should happen is (in order):
1) The number of known positives is calculated using prior year's data
2) That number is subtracted from the total TB or PMTCT_STAT
3) The number of positives (as calculated in another tab) is also subtracted
4) Then the remainder are negatives. 

In other words the "remainder" should default to negative; right now, the remainder defaults to known positive

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g., iOS]
- Excel version [e.g., Office 365, Microsoft 2013, etc.]

**Additional context**
